packages/wallet/manager/useCreateAccounts.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from 'react';
+import { extractAddressFromPassphrase } from '@wallet/utilities/account';
+import { generatePassphrase } from '@views/utilities/passphrase';
+
+const useCreateAccounts = () => {
```

Comment:
Should this be in the `hooks` directory?

Reply:
yhea